src/components/promotion/PromotionCard.tsx

```diff
@@ -23,17 +23,45 @@ const styles: { [identifier: string]: CSSProperties } = {
   },
 };
 
-export default function PromotionCard({
-  promotion,
-  onClick,
-}: {
-  promotion: Promotion;
-  onClick: () => void;
-}): ReactElement {
+interface Props {
+  dateAdded: string;
+  description: string;
+  expirationDate: string;
+  id: string;
+  image: PromotionImageType;
+  name: string;
+  placeId: string;
+  restaurantName: string;
+  savedByUser: boolean;
```

Comment:
do we want to rename the prop to `isSavedByUser`? That way we make it consistent with the `Promotion` interface